integrationTest/test/jest-react/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "react-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "postinstall": "install-local",
+    "test": "stryker run",
+    "posttest": "cross-env TS_NODE_PROJECT=verify/tsconfig.json mocha --require ts-node/register verify/*.ts"
```

Comment:
Is there a need for this TS_NODE_PROJECT? I needed it in angular-project, because angular itself had a tsconfig file in the root and `ts-node` didn't want to execute my verify.ts file that way...